lib/was/registrar/register_object.rb

```diff
@@ -30,11 +30,12 @@ def register_object_using_web_service(register_params)
         Rails.logger.debug "Registering an object with params #{register_params}"
         begin
           resource = RestClient::Resource.new(
-            Rails.configuration.service_root, timeout: 300, open_timeout: 60
+            Rails.configuration.service_root, read_timeout: 300, open_timeout: 60
           )
-          response = resource.post(register_params)
+          # we explicitly want text response so the body is only the druid
+          # New Rails dor-services-app defaults to json; old Sinatra one defaulted to text
```

Comment:
i think we should create a ticket for dor-services-app to maintain the previous behavior, unless it was changed for some specific reason?